pkg/util/secret.go

```diff
@@ -27,7 +32,7 @@ func GetTargetSecret(client kubeclient.Interface, secretReferences []corev1.Obje
 		}
 	}
 
-	return nil, fmt.Errorf("no specific secret found in namespace: %s", targetNamespace)
+	return nil, fmt.Errorf("no specific secret found in namespace: %s: %w", targetNamespace, errNotFound)
```

Comment:
this error should be and only be `errNotFound`? should not contains other informations.

Reply:
or the outer callers never reach IsNotFound branch?

Reply:
Actually both are ok, updated.